commcare_connect/opportunity/views.py

```diff
@@ -86,6 +89,14 @@ def get_form_kwargs(self):
     def form_valid(self, form: OpportunityCreationForm) -> HttpResponse:
         response = super().form_valid(form)
         create_learn_modules_and_deliver_units.delay(self.object.id)
+        interval = IntervalSchedule.objects.get_or_create(every=1, period=IntervalSchedule.DAYS)
```

Comment:
Is this the recommended way to do periodic tasks? I would have imagined it was simpler to make one task and have it iterate over all opportunities, but I am not sure how to schedule that. In HQ we would use the `@periodic_task` decorator and give it a schedule. The other advantage of one task rather than one per domain is that it could quickly end up a very large number of tasks and clog celery.

Reply:
+1 for the one task approach. You can create the schedule with a Django migration or using the `post_migrate` signal.

Reply:
Added a migration to create the periodic task. https://github.com/dimagi/commcare-connect/pull/181/commits/b79b62c8e555eb0a51e7ec5c8e7d224dbbc5cea9